packages/files/directory.pony

```diff
@@ -22,6 +22,10 @@ class Directory
   Capsicum.
   """
   let path: FilePath
+    """
+    The filesystem path pointing to this directory.
```

Comment:
I think we should emphasize here that the `FilePath` type is an object capability that grants access to *do* stuff in this directory - it's more than just a string with a path.